lib/java_buildpack/framework/luna_security_provider.rb

```diff
@@ -144,20 +148,27 @@ def write_configuration(servers, groups)
 VirtualToken = {
 EOS
           groups.each_with_index { |group, index| write_group f, index, group }
-          write_epilogue f
+          write_epilogue f, groups
         end
       end
 
-      def write_epilogue(f)
+      def write_epilogue(f, groups)
         f.write <<EOS
 }
 
 HAConfiguration = {
   AutoReconnectInterval = 60;
   HAOnly = 1;
-  ReconnAtt = 20;
+  reconnAtt = -1;
```

Comment:
The more important change is ReconnectAtt to reconnectAtt as the default value is 0 which means it will never attempt to reconnect (setting is case sensitive).  I will bounce your concerns about having it retry forever off a few people to get some more opinions.